style.css

```diff
@@ -0,0 +1,1226 @@
+body {
+  max-width: 100vw;
+  font-family: arial, sans-serif, serif;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.section-article-heading {
+  display: none;
+}
+
+.tnw-logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 106px;
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+}
+
+.story-background {
+  background-size: 100% 100%;
+  position: absolute;
+  top: 4px;
+  left: 4px;
+  right: 4px;
+  bottom: 4px;
+  z-index: -20;
+}
+
+.story1-background {
+  background: url("images/cover-story1-img.jpg") no-repeat;
+  background-size: cover;
+}
+
+.story2-background {
+  background: url("images/cover-story2-img.jpg") no-repeat;
+  background-size: cover;
+}
+
+.story3-background {
+  background: url("images/cover-story3-img.png") no-repeat;
+  background-size: cover;
+}
+
+.gradient-background {
+  display: block;
+  position: absolute;
+  top: 4px;
+  left: 4px;
+  right: 4px;
+  bottom: 4px;
+  z-index: 1;
+  background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0.25), rgba(0, 0, 0, 1));
```

Comment:
- [ ] This class has almost the same properties as the story-background. Remember that your code should be DRY, so put both classes together and add the different properties after.

Reply:
done